src/Resources/config/listener.yml

```diff
@@ -0,0 +1,89 @@
+##
+# Listener priorities
+#
+# The priorities of the following listeners must be lower than the one of the
+# Symfony route listener (defaults to 32).
+#
+# - 30: ContainerScopeListener
+# - 28: RefererIdListener
+# - 26: InitializeSystemListener
+# - 24: ToggleViewListener
+# - 22: OutputFromCacheListener
+#
+# Session listener
+#
+# The priority of the session listener must be higher than the other listeners but
+# lower than the one of the Symfony session listener (defaults to 128).
+#
+# - 64: SessionListener
+#
+# Do not change the priorities unless you know what you are doing!
+##
+services:
+    contao.listener.add_to_search_index:
+        class: Contao\CoreBundle\EventListener\AddToSearchIndexListener
+        tags:
+            - { name: kernel.event_listener, event: kernel.terminate, method: onKernelTerminate }
+
+    contao.listener.container_scope:
+        class: Contao\CoreBundle\EventListener\ContainerScopeListener
+        arguments:
+            - "@service_container"
+        tags:
+            - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 30 }
+            - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
```

Comment:
Can we be sure that it is the last one when the priority is -254? Symfony defines things with -1000 if I remember correctly.